### PR TITLE
Remove logo from readme file

### PR DESCRIPTION
**Pull request by peteryeungtt**
This is created to remove the logo from readme file

### CODE REVIEW

**Review by peteryeungtt — COMMENTED** [review]
Hi John,

This is Peter, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

## Highlights

- Nice one setting up linters.
- Nice implementing all forms

## Required Changes ♻️

Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

File: index.html [hunk 1]
```diff
@@ -1,37 +1,47 @@
...
+        placeholder="Enter Number"
+      />
+      <label for="" id="name-label">Name</label>
+      <label for="" id="email-label">Email</label>
+      <label for="" id="number-label">Number</label>
+      <select name="" id="dropdown">
+        <option value="">Tall</option>
+        <option value="">Short</option>
+      </select>
+      <input type="radio" name="Choose" id="" value="first" />
```
Comment (peteryeungtt) [inline index.html:39]: For best practices, I suggest use HTML5 semantic.The semantic HTML tags help the search engines and other user devices to determine the importance and context of web pages. The pages made with semantic elements are much easier to read. It has greater accessibility. It offers a better user experience.

File: README.md [hunk 2]
```diff
@@ -25,11 +25,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="/images/snippet.PNG" alt="logo" width="140"  height="auto" />
-  <br/>
-
   <h3><b>My Portfolio Project - README</b></h3>
```
Comment (peteryeungtt) [inline README.md:29]: I suggest adjusting your Readme file to the project, it is unprofessional to use a readme file from a different project without making the necessary changes to it.

**Review by peteryeungtt — COMMENTED** [review]
image

There is a linter error concerning your CSS file that need to be fixed

File: index.html [hunk 3]
```diff
@@ -1,37 +1,47 @@
...
+      <input type="radio" name="Choose" id="" value="second" />
+      <textarea name="" id="" cols="30" rows="10"></textarea>
+      <input type="checkbox" value="1" />
+      <input type="checkbox" value="1" />
+      <button id="submit"></button>
```
Comment (peteryeungtt) [inline index.html:44]: I suggest toadd two spaces of indentation in your. This will make your code looks clean and  more readable.

**Review by Shisui6 — APPROVED** [review]
Hi @peteryeungtt, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good summary ✅
- Good use of Githubs inline comments ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

- It would be nice if you could also add emojis to your inline comments. This makes your feedback look more human friendly👍

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**